client/src/components/Navbar/Notification/Notifications.tsx

```diff
@@ -0,0 +1,124 @@
+import { useRef, useState } from 'react';
+import {
+  Avatar,
+  Button,
+  ListItemAvatar,
+  ListItemText,
+  Menu,
+  MenuItem as DropdownMenuItem,
+  Typography,
+} from '@mui/material';
+import { Box } from '@mui/system';
+import { useStyles } from './useStyles';
+import { NavLink } from 'react-router-dom';
+import io from 'socket.io-client';
+import getUnreadNotifications from '../../../helpers/APICalls/getUnreadNotifications';
+import { Notification as INotification } from '../../../interface/Notification';
+import markNotificationAsRead from '../../../helpers/APICalls/markNotificationAsRead';
+
+const Notification: React.FC<{
+  notif: INotification;
+  handleClose: () => void;
+}> = ({ notif, handleClose }) => {
+  const classes = useStyles();
+
+  const date = new Date(notif.date);
+  const [month, day, year] = [date.getMonth(), date.getDate(), date.getFullYear()];
+
+  const route = notif.type === 'booking' ? '/bookings' : notif.type === 'user' ? '/messages' : {};
```

Comment:
A bit odd approach, ideally this data is attached on the notification itself but for times sake, this is fine for now.